cmd/node/config/config.toml

```diff
@@ -383,6 +383,10 @@
         SameSourceRequests = 10000
         # SameSourceResetIntervalInSec time frame between counter reset, in seconds
         SameSourceResetIntervalInSec = 1
+        # EndpointsThrottlers represents a map for maximum simultaneous go routines for an endpoint
+        EndpointsThrottlers = [{ Endpoint = "/transaction/get", MaxNumGoRoutines = 10 },
```

Comment:
Might lower these values to 2

Reply:
changed the `send` endpoint to support `2` but left the "get" endpoint value to `10` as it was before

Reply:
ok :+1: